docs/configuration.html

```diff
@@ -143,7 +143,7 @@
     <tr>
       <td>rebalance.max.retries</td>
       <td colspan="1">4</td>
-      <td>When a new consumer joins a consumer group the set of consumers attempt to "rebalance" the load to assign partitions to each consumer. If the set of consumers changes while this assignment is taking place the rebalance will fail and retry. This setting controls the maximum number of attempts before giving up.</td>
+      <td>When a Java consumer joins a consumer group the set of consumers attempt to "rebalance" the load to assign partitions to each consumer. If the set of consumers changes while this assignment is taking place the rebalance will fail and retry. This setting controls the maximum number of attempts before giving up.</td>
```

Comment:
Sorry for all the mistaken find/replaces; and thanks for catching them. Will fix those in the next update.